### DISCUSSION TITLE
Where are qemu binaries stored which the app needs?

### DISCUSSION THREAD

**Discussion by GavinPacini** [utmapp/UTM#2425] [UTM Help (macOS)] [answered]
I want to manually run a `qemu-system-aarch64` command, but not sure where to get this binary. I need the one that UTM uses as it has the patches, but rather than cloning and patching myself surely I can find where UTM has put this on my machine. Thanks!

**Comment by conath** [comment, marked as answer]
UTM doesn't include regular QEMU binaries, instead it uses QEMU as a library. 
You'll have to install QEMU on your machine in another way and use the export arguments feature to make your launch script. https://github.com/utmapp/UTM/issues/2280

**Reply by GavinPacini** [reply]
Okay, thanks for the reply!
